consensus/src/messages/mod.rs

```diff
@@ -138,7 +138,7 @@ impl Topic for BlockHeaderTopic {
     const BUFFER_SIZE: usize = 16;
     const NAME: &'static str = "block-header";
     const VALIDATE: bool = true;
-    const MAX_MESSAGES: u32 = 50;
+    const MAX_MESSAGES: u32 = 10;
```

Comment:
Why does `10` mean one per second?

Reply:
`TIME_WINDOW` is 10 seconds.

Reply:
So this is not 1 per second it is 10 per 10 seconds. After a macro block or a skip block this should still work out decently okay. I would still think it should not be as close a constant as it is, but it does not need to be 50. 20 as @hrxi suggested seems good to me.

Reply:
This is per-peer ID, so the chances of the same validator being chosen to produce multiple blocks within a 10-second time window are rather low, so the limit still allows for a bit of a buffer.

Reply:
After checking with @hrxi, the limiting is not per peer id it is a propagation source. We are better off relaxing the max messages to 20 as suggested. For the requesting missing blocks concerns, issue #3071 was created.